scala/lambdas/dynamo-formatters/src/main/scala/uk/gov/nationalarchives/dynamoformatters/DynamoFormatters.scala

```diff
@@ -205,7 +205,7 @@ object DynamoFormatters {
       assetId: ValidatedAttribute[UUID],
       groupId: ValidatedAttribute[String],
       message: ValidatedAttribute[String],
-      potentialCreatedAt: Option[String]
+      CreatedAt: ValidatedAttribute[String]
```

Comment:
The C is capitalised here

Reply:
Arghh ... the perils of just deleting "potential" prefix